rocketchat_API/rocketchat.py

```diff
@@ -297,6 +297,10 @@ def users_update(self, user_id, **kwargs):
         """Update an existing user."""
         return self.__call_api_post("users.update", userId=user_id, data=kwargs)
 
+    def users_set_active_status(self, user_id, user_status, **kwargs):
```

Comment:
Can you please rename it to `active_status`?
Since we don't want to write extra documentation we want everyone to be able to use almost exactly the same parameters as documented by the API devs. This only excludes changes in the style (Python uses snake case instead of camel case) and changes where the variable collides with a python identifier (like `pass`)

Reply:
Do you mean, rename to users_active_status ?
Why don't we put "set" like in users_set_avatar.
If we look at API documentation, the methode is setActiveStatus
https://docs.rocket.chat/api/rest-api/methods/users/setactivestatus

Reply:
Oh, I should have be been more specific. What I mean is the parameter `user_status` on the function should be called `active_status` because in the API is called `activeStatus`.